commands/box/contract/contract.go

```diff
@@ -594,6 +560,13 @@ func call(cmd *cobra.Command, args []string) {
 		params []string
 		err    error
 	)
+	//default abi
+	if err := util.FileExists(abiIdxFile); err != nil {
+		if err := setDefaultAbi(); err != nil {
```

Comment:
if abi idx is not set, report error and return.
It is not proper to set default abi.